operations/charge.go

```diff
@@ -61,18 +62,46 @@ type CreateCharge struct {
 	ReturnURI   string `query:"return_uri"`
 }
 
-func (req *CreateCharge) Op() *internal.Op {
-	op := &internal.Op{
-		Endpoint: internal.API,
-		Method:   "POST",
-		Path:     "/charges",
-		Values:   url.Values{},
+func (req *CreateCharge) MarshalJSON() ([]byte, error) {
+	param := map[string]interface{}{}
+
+	if req.Customer != "" {
+		param["customer"] = req.Customer
```

Comment:
@iporsut Can we use `omitempty` tag ?

Reply:
@chakrit Yes for `Customer, Card, Source, Offsite, ReturnURI` but `dontCapture` can't do that because empty of `bool` is false but we want to can set `false` to `dontCapture` field and I don't want to set this field to `pointer of bool`.